f5lbaasdriver/v2/bigip/service_builder.py

```diff
@@ -465,32 +468,26 @@ def get_db_healthmonitor():
     @log_helpers.log_method_call
     def _get_members(self, context, loadbalancer, pools,
                      subnet_map, network_map):
-        pool_members = []
 
         def get_db_members():
-            if cfg.CONF.f5_driver_perf_mode in (1, 3):
-                members = []
-                for p1 in loadbalancer.pools:
-                    for p2 in pools:
-                        if p1.id == p2['id']:
-                            LOG.info('pool id here:')
-                            LOG.info(p1.id)
-                            members.extend([m for m in p1.members])
-                LOG.info('members right here:')
-                LOG.info(members)
-                return members
-            else:
-                return self.plugin.db.get_pool_members(
-                    context,
-                    filters={'pool_id': [p['id'] for p in pools]}
-                )
+            # normally, there is only one pending updated pool in
```

Comment:
Need to think it carefully. If we are not able to reduce the scope of this performance impact, it will by terrible.